### PR TITLE
Hardening/117 put subscription conv op

### PR DESCRIPTION
**Pull request by kzangeli**
Function header comments and functest for:
PUT /v1/contextSubscriptions/{subscriptionId}

### CODE REVIEW

File: test/functionalTest/cases/117_convop_using_standard_ops/putSubscriptionConvOp.test [hunk 1]
```diff
@@ -0,0 +1,763 @@
...
+
+
+echo "06. Dump accumulator, see E1/T1/A1+A2, in XML"
+echo "============================================="
+curl localhost:${LISTENER_PORT}/dump -s -S 2> /dev/null
+echo
+echo
+
+
+echo "07. PUT /v1/contextSubscriptions/SUB_ID?notifyFormat=JSON, removing A2 from notifyConditions"
```
Comment (fgalan) [inline test/functionalTest/cases/117_convop_using_standard_ops/putSubscriptionConvOp.test]: noityFormat makes sense when you want to change the default notification format (which is the one used in the subscription payload). Thus, in this case, given that the payload is in JSON, shouldn't be `?notifyFormat=XML`?

Comment (kzangeli) [inline test/functionalTest/cases/117_convop_using_standard_ops/putSubscriptionConvOp.test]: Actually, whenever the notifyFormat parameter is used, it overrides the 'default' which is whatever formats the request was sent in. It pretty much always makes sense. If notifyFormat is present, never mind the format of the request.

Comment (fgalan) [inline test/functionalTest/cases/117_convop_using_standard_ops/putSubscriptionConvOp.test]: What I mean is to use set notifyFormat to the opposite format of the request to ensure it is actually working (if you use JSON payload and notifyFormat=JSON to ckeck you get JSON notification, you are not actually checking it works).

If my point is not clear, let's move discussion to skype.

Comment (kzangeli) [inline test/functionalTest/cases/117_convop_using_standard_ops/putSubscriptionConvOp.test]: It is pretty clear ... :-)
?notifyFormat=XML 'the opposite way' is already used in 'part 04'.
Here, in 'part 07', I can either use it or not, doesn't really make a difference ...

Comment (fgalan) [inline test/functionalTest/cases/117_convop_using_standard_ops/putSubscriptionConvOp.test]: I didn't realize that part 04 works in cross-format way (text: "04. Subscribe to E1/T1/A1+A2, in XML" doesn't mention it and I usually only pay attention to the test outline). Thus, I think is ok as it is.

NTC

**Comment by fgalan** [comment]
LGTM

**Pull request merged**